sdk/easm/azure-analytics-defender-easm/README.md

```diff
@@ -0,0 +1,128 @@
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authentication
+
+[Azure Identity][azure_identity] package provides the default implementation for authenticating the client.
+
+## Key concepts
+### [Assets][assets_documentation]
+Defender EASM includes the discovery of the following kinds of assets:
+-   Domains
+-   Hosts
+-   Pages
+-   IP Blocks
+-   IP Addresses
+-   Autonomous System Numbers (ASNs)
+-   SSL Certificates
+-   WHOIS Contacts
+
+These asset types comprise your attack surface inventory in Defender EASM. This solution discovers externally facing assets that are exposed to the open internet outside of traditional firewall protection; they need to be monitored and maintained to minimize risk and improve an organization’s security posture. Microsoft Defender External Attack Surface Management (Defender EASM) actively discovers and monitors these assets, then surfacing key insights that help customers efficiently address any vulnerabilities in their organization.
+
+### [Discovery][discovery_documentation]
+Microsoft Defender External Attack Surface Management (Defender EASM) relies on our proprietary discovery technology to continuously define your organization’s unique Internet-exposed attack surface. Discovery scans known assets owned by your organization to uncover previously unknown and unmonitored properties. Discovered assets are indexed in a customer’s inventory, providing a dynamic system of record of web applications, third party dependencies, and web infrastructure under the organization’s management through a single pane of glass.
+
+#### [Discovery Groups][discovery_groups_documentation]
+Custom discoveries are organized into Discovery Groups. They are independent seed clusters that comprise a single discovery run and operate on their own recurrence schedules. Users can elect to organize their Discovery Groups to delineate assets in whatever way best benefits their company and workflows. Common options include organizing by responsible team/business unit, brands or subsidiaries.
+
+## Examples
+
+```java com.azure.analytics.defender.easm.ReadmeSamples
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.analytics.defender.easm;
+
+import com.azure.core.util.Configuration;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+
+public final class ReadmeSamples {
+    public void readmeSamples() {
+        // BEGIN: com.azure.analytics.defender.easm.readme
+        String subscriptionId = Configuration.getGlobalConfiguration().get("SUBSCRIPTIONID");
+        String workspaceName = Configuration.getGlobalConfiguration().get("WORKSPACENAME");
+        String resourceGroupName = Configuration.getGlobalConfiguration().get("RESOURCEGROUPNAME");
+        String endpoint = Configuration.getGlobalConfiguration().get("ENDPOINT");
+
+        EasmClient easmClient = new EasmClientBuilder()
```

Comment:
Take a look at the README from App Configuration and add similar samples.
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/appconfiguration/azure-data-appconfiguration/README.md